graph/Makefile

```diff
@@ -1,10 +1,12 @@
 .DEFAULT_GOAL := help
+export UI_STATIC_CONTENT = $(PWD)/tmp
+
 # Porcelain
 # ###############
 .PHONY: env-up env-down env-recreate container run build lint test
 
 run: setup ## run the app
-	python src/main.py <./tmp/fake_graph.json
+	python src/main.py $(UI_STATIC_CONTENT)
```

Comment:
I've talked about an environment variable - but if you have good reasons for doing it this way we can absolutely go with it. Do you?

Reply:
no xD